src/lib/Post.svelte

```diff
@@ -525,18 +567,25 @@
 							);
 						}
 					} catch (e) {
-						editError = e;
+						error = e;
 					}
 				}}>Save</button
 			>
 		</div>
 	{:else}
-		<p class="post-content" style="border-left-color: #4b5563;">
-			{@html addFancyElements(post.content)}
+		<p
+			class="post-content"
+			style="border-left-color: #4b5563; {post.pending
+				? 'color: #A9A9A9'
+				: 'color: #ffffff'}"
```

Comment:
this probably won't work in light mode or other themes (maybe just change the opacity instead)

Reply:
Oops, that is pretty bad. I will fix.